code/API_definitions/Edge-Application-Management.yaml

```diff
@@ -547,6 +547,59 @@ paths:
           $ref: '#/components/responses/500'
         '503':
           $ref: '#/components/responses/503'
+  /clusters:
+    get:
+      security:
+        - openId:
+            - edge-application-management:clusters:read
+      tags:
+        - Cluster
+      summary: Retrieve a list of the available clusters
+      description: |
+        List available cluster information
+      operationId: getClusters
+      parameters:
+        - $ref: '#/components/parameters/x-correlator'
+        - name: region
```

Comment:
As the EdgeCloudZone has been used as the prime factor to refer to edge resources and region as the secondary attribute so should we include the EdgeCloudZone and optionally the region?